android-base/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
This code is for the root directory. So we don't need this file. 🙇

Reply:
Sorry. I can't checked.
delete it